### PR TITLE
1 add landing message

### PR DESCRIPTION
**Pull request by asmyra**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
 <titles>Welcome to Potionfy</titles>
 </head>
 <body>
-
+<h1>Welcome to Potionfy!Enjoy your stay</h1>
```
Comment (asmyra) [inline index.html:10]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

**Comment by nurasmyra** [comment]
Sure, can do!

File: index.html (hunk 1 above)
Comment (nurasmyra) [reply index.html:10]: Sure, can do!

**Review by nurasmyra — COMMENTED** [review]
Sure, can do!

**Pull request merged**